### ISSUE TITLE
int far feclearexcept( int ) not found by linker

### ISSUE THREAD

**Issue by arstrube** [open-watcom/open-watcom-v2#232]
I receive the following linker errors:
`Error! E2028: int far feclearexcept( int ) is an undefined reference ` 
and:
`undefined symbol int far feclearexcept( int )`

It is my understanding that the linker should be able to find the symbol, in the same way as many other library functions I am using. A text search reveals that the string does indeed occur in the libraries, e.g.:
```
  C:\data\_DevTools\watcom\lib386\dos\clib3s.lib (2 hits)
	Line 656: feclearexcept
	Line 3903: feclearexceptí_telli64î
```
Am I missing something obvious? Or is this a bug?

**Comment by arstrube** [comment]
Okay - solved. It wasn't obvious from the error messages, but the call was in C++ code and therefore the header needed to be wrapped in `extern "C" {}`.

**Comment by jmalak** [comment]
Thanks for bug report.
It is bug even if you can simply fix if by extern "C" block over this header file.
Any OW C header file must use guard for C++ compiler.
I will fix it.

**Comment by arstrube** [comment]
Thanks! Much appreciated.

**Comment by jmalak** [comment]
It is fixed by 3db37ba861866d6ec855375ab180ace6cd2d3ce6 in git repository.
It fixes some other minor issues in header files.
